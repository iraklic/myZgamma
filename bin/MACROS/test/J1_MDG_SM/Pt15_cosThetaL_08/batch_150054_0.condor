000 (150054.000.000) 09/26 23:59:38 Job submitted from host: <131.225.191.210:35697>
...
001 (150054.000.000) 09/27 00:00:25 Job executing on host: <131.225.191.142:45019>
...
006 (150054.000.000) 09/27 00:00:34 Image size of job updated: 81240
...
005 (150054.000.000) 09/27 01:58:43 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:57:51, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:57:51, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	108481  -  Run Bytes Sent By Job
	9484435  -  Run Bytes Received By Job
	108481  -  Total Bytes Sent By Job
	9484435  -  Total Bytes Received By Job
...
