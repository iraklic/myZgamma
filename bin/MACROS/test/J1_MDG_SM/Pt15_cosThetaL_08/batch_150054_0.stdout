
CMSSW on Condor

started at Tue Sep 27 00:00:27 CDT 2011

parameter set:
CONDOR_CLUSTER: 150054
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_150054_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_150054_0.log 2>&1\n

finished at Tue Sep 27 01:58:43 CDT 2011
